library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java

```diff
@@ -453,6 +454,35 @@ protected void setDismissListener(DismissListener dismissListener) {
         mDismissListener = dismissListener;
     }
 
+    /**
+     * Returns if current shape has a radius. Have to add more cases if new shapes doesn't have radius
+     *
+     * @return true if current shape has a radius
+     */
+    public boolean hasRadius(){
+
+        if(mFocusShape.equals(FocusShape.CIRCLE))
+            return true;
+
+        return false;
+    }
+
+    /**
+     * Returns mRadius or -1 if mRadius == 0 (i.e. Radius doesn't exist for the shape)
+     *
+     * @return mRadius or -1 if mRadius == 0
+     */
+    public int getFocusRadius(){
```

Comment:
Only returning `mRadius` will be enough.
Also `mRadius` initial value will be `-1`